### DISCUSSION TITLE
Can't create or update response scripts with "shared_with_workflows" set to "True"

### DISCUSSION THREAD

**Discussion by Rob2n** [CrowdStrike/falconpy#1225] [Ideas]
**Describe the bug**
The RTR_CreateScripts feature doesn't allow setting the "Share with workflows" to True (it is by default False).
Not sure if this is a bug or simply a missing feature, apologies if this is the wrong template for this.

**To Reproduce**
```python
falcon.RTR_CreateScripts(
            files=script_upload,
            comments_for_audit_log="",
            description="",
            permission_type="public",
            share_with_workflows=True, # Property in present when getting scripts but we can't set it when creating/updating ?
        )
```

**Expected behavior**
Expected behavior would be to have the possibility of configuring this setting on script creation.

**Environment (please complete the following information):**
 - OS: Windows 10
 - Python: 3.10
 - FalconPy: 1.2.10

**Additional context**
I am deploying a script to many child CIDs using FalconPy and realized it was not possible to set the "Share with workflows" setting to True.
This is greatly reducing the usefulness of the bulk deployment feature for scripts,  and I didn't see any option to set it in the docs.
Thanks in advance for your help

**Comment by jshcodes** [comment]
Hi @Rob2n -

I reached out internally and have confirmed that this functionality is on the roadmap and currently being worked on.  Once the new parameter key hits swagger, we'll update FalconPy (and our other SDKs) to support it.

Thank you for reporting this!
